_actions_/actionsRoute.js

```diff
@@ -2,159 +2,115 @@ const express = require('express');
-//         .then(post => {
-//             console.log(post.user_id)
-//             res.status(200).json(post)
-//         })
-//         .catch(err => {
-//             res.status(404).json({ error: "The User with that ID doesn't exsist" })
-//         })
-// });
-
-// router.delete('/:id', validateUserId, (req, res) => {
-//     const { id } = req.params
-//     db.remove(id)
-//         .then(removed => {
-//             if (removed) {
-//                 res.status(204).end()
-//             } else (
-//                 res.status(404).json({ message: "The User with the specified ID does not exist." })
-//             )
-//         })
-//         .catch(err => {
-//             console.log(err)
-//             res.status(500).json({ error: "The User could not be removed." })
-//         })
-// });
-
-// router.put('/:id', validateUserId, validateUser, (req, res) => {
-//     const { id } = req.params
-//     const { name } = req.body
-
-//     db.update(id, { name })
-//         .then(updated => {
-//             console.log('Line 101', updated)
-//             db.getById(id)
-//                 .then(nameReturned => {
-//                     console.log('Line 104', nameReturned)
-//                     if (nameReturned) {
-//                         res.status(201).json(nameReturned)
-//                     } else {
-//                         res.status(404).json({ error: "The name with the specified ID does not exist." })
-//                     }
-//                 })
-//                 .catch(err => {
-//                     console.log(err)
-//                     res.status(500).json({ error: "There was an error while saving the name to the database" })
-//                 })
-//         })
-//         .catch(err => {
-//             console.log(err)
-//             res.status(500).json({ error: "There was an error while saving the name to the database." })
-//         })
-// });
-
-// //custom middleware
-
-// function validateUserId(req, res, next) {
-//     db.getById(req.params.id)
-//         .then(results => {
-//             if (results) {
-//                 req.user = results
-//                 console.log(req.user)
-//                 next()
-//             } else {
-//                 res.status(404).json({ message: "User ID is invalid" })
-//             }
-//         })
-// };
-
-// function validateUser(req, res, next) {
-//     const user = req.body;
-//     if (!user) {
-//         res.status(400).json({ message: 'Missing user data' })
-//     } else if (!user.name) {
-//         res.status(400).json({ message: 'Missing required name field' })
-//     } else {
-//         next();
-//     }
-// };
-
-// function validatePost(req, res, next) {
-//     const post = req.body
-//     post.user_id = req.params.id
-//     if (!post.text) {
-//         res.status(400).json({ message: "missing required text field" })
+// Posting for a new action requires a project_id that needs to be checked against exsisting project ids
```

Comment:
a little more comments on the logic, otherwise good job increasing how many comments you have